draft-ietf-quic-transport.md

```diff
@@ -1278,6 +1278,9 @@ IDs, QUIC processes the packet as part of that connection. Endpoints MUST drop
 packets with zero-length Destination Connection ID fields if they do not
 correspond to a single connection.
 
+Endpoints SHOULD send a Stateless Reset ({{stateless-reset}}) for any packets
```

Comment:
I think this should be restricted to short-header packets, correct?

Reply:
We already have text in the Stateless Reset section saying an endpoint MAY send a stateless reset for a LH packet.  I can't really understand how an implementation would do that, though.

Reply:
Yes, let's not reiterate that discussion.  Right now, it's basically impossible to except in a very narrow window during the handshake, but we agreed that there was no reason not to allow it.  If the long header is used after the handshake completes (for reasons we come up with later), then we don't have to worry about stateless reset.